client/state/account-recovery/test/reducer.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+
+import {
+	ACCOUNT_RECOVERY_FETCH_SUCCESS,
+} from 'state/action-types';
+
+import dummyData from './test-data';
+import reducer from '../reducer';
+
+describe( 'account-recovery reducer', () => {
+	const camelDummyData = {
```

Comment:
Ahh right. At first I was thinking "hmm, it is just a camel-case keyed version of `dummyData`", hence the name. But it is nonsense, so I renamed it as `expectedState` to better fit the meaning 7f60f8abe6aae8fe7337e3a47a56dc3223771e8d